pkg/util/util.go

```diff
@@ -0,0 +1,14 @@
+package util
+
+import "unicode"
+
+// CapitalizeFirstChar func capitalizes the first character of the string
+func CapitalizeFirstChar(str string) string {
+	if len(str) == 0 {
+		return ""
+	}
+
+	runes := []rune(str)
+	runes[0] = unicode.ToUpper(runes[0])
+	return string(runes)
+}
```

Comment:
**question:** `util` is not a very descriptive package. Can we avoid using util in favor of something more descriptive. `title case` for instance might be a more descriptive way to describe what this package does.